README.md

```diff
@@ -44,12 +44,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check if SCOPE_APIKEY is set
-        run: if [ "${{secrets.SCOPE_APIKEY}}" = "" ]; then exit 1; fi
-      - uses: actions/checkout@v1
-      - name: Run Scope Maven JDK 1.8
-        uses: docker://undefinedlabs/scope-action-maven-jdk18:latest
-        env:
-          SCOPE_APIKEY: ${{secrets.SCOPE_APIKEY}}
+        run: if [ "${{secrets.SCOPE_DSN}}" = "" ]; then exit 1; fi
```

Comment:
Does the action fail if the DSN is not set? If so, we can remove this now, right?

Reply:
Yes, removing it.